pokemonred_puffer/train.py

```diff
@@ -62,15 +62,15 @@ def load_from_config(
 
 def make_env_creator(
     wrapper_classes: list[tuple[str, ModuleType]],
-    reward_class: ModuleType,
+    reward_class: RedGymEnv,
 ) -> Callable[[pufferlib.namespace, pufferlib.namespace], pufferlib.emulation.GymnasiumPufferEnv]:
     def env_creator(
         env_config: pufferlib.namespace,
         wrappers_config: list[dict[str, Any]],
         reward_config: pufferlib.namespace,
     ) -> pufferlib.emulation.GymnasiumPufferEnv:
-        env = reward_class(RedGymEnv(env_config), reward_config)
-        for cfg, (_, wrapper_class) in zip(wrappers_config, wrapper_classes):
+        env = reward_class(env_config, reward_config)
+        for cfg, (_, wrapper_class) in zip(wrappers_config, wrapper_classes.items()):
```

Comment:
I think when you made this change, your code was slightly out of date relative to what is in main. I made some changes last night (such as wrapper_classes is a list instead of a dict) so a user can reuse a wrapper multiple times in their config.

Reply:
Here are two issues, actually. 

```
        env = reward_class(env_config, reward_config)
```
So, I think `reward_class` is `BaselineRewardEnv`, which takes in `env_config` and `reward_config`, and this is not a wrapper class. I was wondering, why it was passing RedGymEnv instance. 

I like the new `wrapper_classes` approach. I encountered an error there, just added `items()` to make zip work.

Reply:
I'm just a little confused how `items` works here if `wrapper_classes` is now a `list`.

Reply:
Oh you are right. I copy pasted an older-version of `setup_agent()` that creates wrapper_classes dictionary. Reverting this back.